packages/DropdownMenu/src/components/Trigger/Trigger.js

```diff
@@ -2,37 +2,42 @@ import React from "react";
 import PropTypes from "prop-types";
 import Button from "@paprika/button";
 
-const handleKeyDown = e => {
-  e.stopPropagation();
-};
-
 const propTypes = {
+  children: PropTypes.node,
+  icon: PropTypes.node,
   isOpen: PropTypes.bool.isRequired,
-  handleOpenMenu: PropTypes.func.isRequired,
-  triggerRef: PropTypes.shape({ current: PropTypes.instanceOf(Object) }) || null,
   menuRefId: PropTypes.string,
+  onOpenMenu: PropTypes.func.isRequired,
+  triggerRef: PropTypes.shape({ current: PropTypes.instanceOf(Object) }) || null,
 };
 
-const defaultProps = { triggerRef: null, menuRefId: "" };
+const defaultProps = {
+  children: null,
+  icon: null,
+  menuRefId: "",
+  triggerRef: null,
+};
 
 const Trigger = props => {
-  const { isOpen, handleOpenMenu, menuRefId, triggerRef, ...otherProps } = props;
+  const { icon, children, isOpen, onOpenMenu, menuRefId, triggerRef, ...otherProps } = props;
+  const TriggerComponent = icon ? Button.Icon : Button;
   return (
-    <Button
+    <TriggerComponent
       ref={triggerRef}
       aria-controls={menuRefId}
       aria-expanded={isOpen}
       aria-haspopup="true"
-      onClick={handleOpenMenu}
-      onKeyDown={handleKeyDown}
+      onClick={onOpenMenu}
       isSquare
       isSemantic={false}
       {...otherProps}
-    />
+    >
+      {icon || children}
+    </TriggerComponent>
   );
 };
 
+Trigger.displayName = "DropdownMenu.Trigger";
```

Comment:
This is basically a copy of `Confirmation.Trigger`? I've heard of WET code, but this makes my spidey sense tingle.  We should at least create an issue to consolidate all of these `Trigger` components...